starlette/graphql.py

```diff
@@ -26,9 +26,11 @@ def __init__(
         executor: typing.Any = None,
         executor_class: type = None,
         graphiql: bool = True,
+        middleware: list = None,
     ) -> None:
         self.schema = schema
         self.graphiql = graphiql
+        self.middleware = middleware
```

Comment:
Missing coma at the end of line. 

Using comas even in the last entry (dict/list/tuples/function args/etc.) is nice to avoid having diffs in the future when we add a line right after.

For example: 

```
d = {
    'x': 1, 
    'y': 2
}
```

Then someone adds a new entry in the next commit diff you would have:

```diff
d = {
    'x': 1, 
+   'y': 2,
+   'z': 3
}
```

I'm pretty sure that's the reason why black enforces such a rule.

Reply:
Hello, 

This should be fixed now.
I've added the commas using the black formatter. :)

Reply:
I think line 33 should not have a comma since it's not defining an argument but simply set the middleware of the object from the argument.

Reply:
Right, my bad. That's why automatic checks are useful 👍.